docs/source/tutorial/glance.rst

```diff
@@ -0,0 +1,427 @@
+Chainer at a Glance
+===================
+
+.. currentmodule:: chainer
+
+Welcome to Chainer!
+
+Chainer is a rapidly growing neural network platform. The strengths of Chainer are:
+
+* 100% pure Python -- Chainer is developed from the entirely in Python, allowing for inspection and customization of all code in python and understandable python messages at run time
+* Define by Run -- neural networks definitions are defined on-the-fly at run time, allowing for dynamic network changes
+* Fully customizable -- since Chainer is pure python, all classes and methods can be adapted to allow for the latest cutting edge or specialized approaches
+* Broad and deep support -- Chainer is actively used for most of the current approaches for neural nets (CNN, RNN, RL, etc.), aggressively adds new approaches as they're developed, and provides support for many kinds of hardware as well as parallelization for multiple GPUs
+
+
+Mushrooms -- tasty or deathly?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's take a look at a basic program of Chainer to see how it works. For a dataset, we'll work with `Kaggle's edible vs. poisonous mushroom dataset <https://www.kaggle.com/uciml/mushroom-classification>`_, which has over 8,000 examples of mushrooms, labelled by 22 categories including odor, cap color, habitat, etc.
+
+How will Chainer learn which mushrooms are edible and which mushrooms will kill you? Let's see!
+
+Full Code
+---------------
+
+Here's the whole picture of the code:
+
+.. testcode::
+    
+   #!/usr/bin/env python
+   
+   from __future__ import print_function
```

Comment:
This is not essential as a tutorial. How about fixing `print` and removing this import?